admin/spec/components/previews/solidus_admin/ui/badge/component_preview.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# @component "ui/badge"
+class SolidusAdmin::UI::Badge::ComponentPreview < ViewComponent::Preview
+  layout "solidus_admin/preview"
+  include SolidusAdmin::ContainerHelper
+
+  # @param name [String]
+  def overview(name: "Label")
+    render_with_template(locals: { name: name, component: component("ui/badge") })
```

Comment:
If we add the `BaseComponentPreview`, maybe we can extract that `component("ui/badge")` into a helper method `component`? We can do it later, though.

Reply:
Good call! You think would be worth it having a local helper with some duplication assuming we won't be able to extract the superclass?

Reply:
I'm not sure I follow... if we had the `BaseComponentPreview` (which it looks it's not that easy) we should be able to extract the component from the preview class name, right? `class SolidusAdmin::UI::Badge::ComponentPreview` => `ui/badge`

Reply:
Yes, if we had the BaseComponentPreview that would be super easy, I was wondering if would make sense to have it in every preview class while we don't have the base class. Sorry about the confusion.

Reply:
Yeah, that works if the base class is not an option.